src/usr/bin/google_set_multiqueue

```diff
@@ -124,6 +124,48 @@ function is_gvnic() {
   return 0
 }
 
+# Returns the vCPU ranges on each of the numa nodes. The vCPU ranges will
+# be in the form of array of
+# [numa0_irq_start_1, numa0_irq_end_1, numa0_irq_start_2, numa0_irq_end_2,
+# numa1_irq_start_1, numa1_irq_end_1, numa1_irq_start_2, numa1_irq_end_2]
+function get_vcpu_ranges {
```

Comment:
Please name this something that `get_a3_vcpu_ranges` so that the assumptions and intended usage are a little clearer

Reply:
renamed to `get_vcpu_ranges_on_accelerator_platform`